packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -191,3 +186,86 @@ function isSecondsPropertyMisusedInCalendarTriggerInput(
       Object.keys(triggerWithoutChannelId).length > 1)
   );
 }
+
+function validateDateComponents(
+  object: any,
```

Comment:
```suggestion
  trigger: NotificationTriggerInput,
```

Reply:
yes, agree, considering the context this is better 👍